set-matter.sh

```diff
@@ -71,7 +76,17 @@ if [ "$UID" -eq "$ROOT_UID" ]; then
       echo -e "Setting theme to ${PALETTE}"
       sed -i -E "s/(selected_item_color = )\"#[0-9a-fA-F]+\"/\1\"${!PALETTE}\"/" "${THEME_DIR}/${THEME_NAME}/theme.txt"
   fi
-  
+
+  # Setting available fonts
+  # List of supported fonts
+  supported_fonts=(poiret-one karla pattaya)
+
+  # Set the chosen font
+  if [[ ! " ${FONT} " == "  " ]] && [[ " ${supported_fonts[@] " =~ " ${FONT} " ]]; then
```

Comment:
${supported_fonts[@]}
right curly brace is missing

Reply:
thanks, will fix it in the next commit